src/main/java/frc/robot/commands/Drive/ArcadeDrive.java

```diff
@@ -56,6 +61,18 @@ public void execute() {
       } else if (rotation > 0) {
         rotation = rotation - Constants.XBOX_DEADBAND_LIMIT;
       }
+
+      //Demo Mode
+        double percentSpeed = m_drive.getPercentPower();
+        
+        speed = speed * percentSpeed;
```

Comment:
Indentation for lines 66 through 75 needs to be fixed. The lines are indented by an extra two spaces.